src/components/breadcrumb/README.md

```diff
@@ -13,3 +13,24 @@ Setting this to `display: none` will remove the associated slash.
   display: none;
 }
 ```
+
+
+
+Add `.bx--breadcrumb--no-trailing-slash` to `.bx--breadcrumb` to remove leading slashes. 
```

Comment:
Good catch... it was update in my local but forgot to commit again... Thanks... @hellobrian